tox.ini

```diff
@@ -1,29 +1,30 @@
 [tox]
 envlist =
     # Python/Django combinations that are officially supported
-    py{27,35,36,37}-django111
-    py{35,36,37,38}-django22
-    py{36,37,38}-django30
+    py{36,37,38,39}-django22
+    py{36,37,38,39}-django30
+    py{36,37,38,39}-django31
     py37-{flake8,bandit,readme,docs}
 
 [testenv]
 description = Unit tests
 deps =
     coverage
-    django111: Django>=1.11,<2.0
     django22: Django>=2.2,<3.0
     django30: Django>=3.0,<3.1
+    django31: Django>=3.1,<3.2
+
 commands =
     coverage run setup.py test
     coverage xml
 
 [gh-actions]
 python =
-    2.7: py27
     3.5: py35
```

Comment:
`py35` should probably be removed if we want this to match the trove classifiers in `setup.py`.